core/src/main/java/io/confluent/rest/metrics/MetricsResourceMethodApplicationListener.java

```diff
@@ -189,6 +190,14 @@ public MethodMetrics(ResourceMethod method, PerformanceMetric annotation, Metric
 
       this.requestSizeSensor = metrics.sensor(getName(method, annotation, "request-size"));
       MetricName metricName = new MetricName(
+          getName(method, annotation, "request-count-windowed"), metricGrpName,
+          "The request count using a windowed counter", metricTags);
+      this.requestSizeSensor.add(metricName, new WindowedCount());
+      metricName = new MetricName(
+          getName(method, annotation, "request-count-cumulative"), metricGrpName,
+          "The request count using a cumulative counter", metricTags);
+      this.requestSizeSensor.add(metricName, new WindowedCount());
```

Comment:
Should this be CumulativeCount?

Reply:
Oh, good spot!

Reply:
I have updated the unit tests to spot this too.